### PR TITLE
Making the repo workable and fixed the bugs

### PR DESCRIPTION
**Pull request by PuravG**
i merged all the pull requests and made the repo runnable (on my end atleast)

### CODE REVIEW

File: target/classes/map.html
Comment (mairara) [inline target/classes/map.html:1]: did you mean to delete this whole html?

File: .idea/.gitignore [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
+.idea/misc.xml
 
 # Ignore compiled Java class files
 *.class
 
 # Ignore build directories
-/out/
-/target/
-/build/
+out/
```
Comment (mairara) [inline .idea/.gitignore:13]: omg that was the problem !

**Review by mairara — CHANGES_REQUESTED** [review]

File: target/classes/map.html
Comment (PuravG) [reply target/classes/map.html:1]: Uhh I think so, bcos it was in the target class and I think that might have been auto generated but also I'm not too sure.

Comment (mairara) [reply target/classes/map.html:1]: hmm okay i think it should be okay to delete then. I thought it was something to do with ur use case lol.

**Review by mairara — APPROVED** [review]
Merging now!

**Pull request merged**